File: fish_nf/lib/src/platform_specifics/android/styles/beauty_style_information.dart
import 'default_style_information.dart';

class BeautyStyleInformation extends DefaultStyleInformation {
  const BeautyStyleInformation(this.title, this.body, this.image, this.button, this.appIcon)
      : super(false, false);

  final String title;
  final String body;
  final String image;
  final String button;
  final String appIcon;
}

File: fish_nf/android/src/main/java/com/dexterous/flutterlocalnotifications/models/styles/BeautyStyleInformation.java
package com.dexterous.flutterlocalnotifications.models.styles;

import androidx.annotation.Keep;
import androidx.annotation.NonNull;

@Keep
public class BeautyStyleInformation extends DefaultStyleInformation {
    @NonNull
    public String title;
    @NonNull
    public String body;
    @NonNull
    public String image;
    @NonNull
    public String button;
    @NonNull
    public String appIcon;

    public BeautyStyleInformation(@NonNull String title, @NonNull String body, @NonNull String image,
                                  @NonNull String button, @NonNull String appIcon) {
        super(false, false);
        this.title = title;
        this.body = body;
        this.image = image;
        this.button = button;
        this.appIcon = appIcon;
    }
}
